.github/dependabot.yml

```diff
@@ -1,8 +1,14 @@
 version: 2
 updates:
-- package-ecosystem: npm
-  directory: "/"
-  schedule:
-    interval: daily
-    time: "16:00"
-  open-pull-requests-limit: 10
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: weekly
+      time: "16:00"
+    open-pull-requests-limit: 10
```

Comment:
I'm also inclined to decrease this limit down to 5 - if you agree

Reply:
I thought about that too, but I was wondering whether it would lead to updates getting "backed up"

Reply:
we can leave it for now, but i would be curious - dependabot could choose the "most important 5" every week, or something

alternatively - we could run frequently, but have a `open-pull-requests-limit: 1` - and therefore dependabot must finish and merge each dependency before opening a new PR. this might reduce the huge backup of Action jobs?

Reply:
one of the big problems right now - is when one gets merged - the other 9 all get rebased and spin up all their jobs at once - 8 of which will not have value

Reply:
ahh, true, we do have it set up to get merged automatically!

You're right, it would be interesting to try that. Might be a better solution.

Reply:
we could try `daily` & `-limit: 1` out on this repo and see how it goes

Reply:
Yes please! I think that would be a good experiment.

Reply:
alright I changed it to `daily` with limits of 1. Seems like we can't have a global limit; each package ecosystem has to have its own limit (https://github.com/dependabot/dependabot-core/issues/2158). This shouldn't be too bad because the `github-actions` one is low volume I think.